tests/unit_10.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test, expect } from '@playwright/test';
+
+test('Test 1 - Verify login with valid credentials', async ({ page }) => {
+  await page.goto('https://practicesoftwaretesting.com/auth/login');
```

Comment:
I'd move the url to the config file https://playwright.dev/docs/test-use-options#basic-options